packages/aws-cdk/integ-tests/test-cdk-multiple-toolkit-stacks.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -euo pipefail
+scriptdir=$(cd $(dirname $0) && pwd)
+source ${scriptdir}/common.bash
+# ----------------------------------------------------------
+
+setup
+
+toolkit_stack_name_1="toolkit-stack-${RANDOM}"
+toolkit_stack_name_2="toolkit-stack-${RANDOM}"
```

Comment:
There is a small, but non-zero, probability that the two randoms will be equal. Maybe this just annoys me, but I'd generate a new `toolkit_stack_name_2` until it is not equal to `toolkit_stack_name_1`. Another option is to actually use a different string literal portion in each...

Reply:
Fair enough, added `-1` and `-2` suffixes to prevent this case